docs/integrations/amazon-aws/sqs.md

```diff
@@ -13,235 +13,221 @@ Amazon Simple Queue Service (Amazon SQS) is a fully managed message queuing serv
-   "eventName":"CreateQueue",
-   "awsRegion":"us-west-2",
-   "sourceIPAddress":"19.174.45.8",
-   "userAgent":"aws-sdk-go/1.12.8 (go1.9; darwin; amd64) APN/1.0 HashiCorp/1.0 Terraform/0.10.0-dev",
-   "requestParameters":{
-      "queueName":"msg_process_businessrule",
-      "Attribute":{
-         "ReceiveMessageWaitTimeSeconds":"20",
-         "MessageRetentionPeriod":"345600",
-         "MaximumMessageSize":"262144",
-         "VisibilityTimeout":"3600"
-      }
-   },
-   "responseElements":{
-      "queueUrl":"https://sqs.us-west-2.amazonaws.com/12212341717/initial_msg_formatting"
-   },
-   "requestID":"3f9f0a8a-1234-5678-b16c-58fc1a1ee8fb",
-   "eventID":"66b74ca6-1234-5678-a61e-fba42272ba91",
-   "eventType":"AwsApiCall",
-   "recipientAccountId":"122123451717"
+         "eventTime":"2022-10-19T09:26:18+0000",
+         "eventSource":"sqs.amazonaws.com",
+         "eventName":"SetQueueAttributes",
+         "awsRegion":"us-east-1",
+         "sourceIPAddress":"195.186.216.125",
+         "userAgent":"AWS Internal",
+         "requestParameters":{
+           "attributes":{
+        "Policy":"{\"Version\":\"2008-10-17\",\"Id\":\"__default_policy_ID\",\"Statement\":  [{\"Sid\":\"__owner_statement\",\"Effect\":\"Allow\",\"Principal\":{\"AWS\":\"arn:aws:iam::956882708938:root\"},\"Action\":\"SQS:*\",\"Resource\":\"arn:aws:sqs:us-east-1:956882708938:JayanatTest4\"},{\"Sid\":\"topic-subscription-arn:aws:sns:us-east-1:956882708938:SNSAppSomya01\",\"Effect\":\"Allow\",\"Principal\":{\"AWS\":\"*\"},\"Action\":\"SQS:SendMessage\",\"Resource\":\"arn:aws:sqs:us-east-1:956882708938:JayanatTest4\",\"Condition\":{\"ArnLike\":{\"aws:SourceArn\":\"arn:aws:sns:us-east-1:956882708938:SNSAppSomya01\"}}}]}"
+      },
+      "queueUrl":"https://sqs.us-east-1.amazonaws.com/123456789033/pull_private_submodule_jobs.fifo"
+         },
+         "responseElements":null,
+         "requestID":"635ae9dd-83cc-5b42-890e-b273c168cb35",
+         "eventID":"874213f3-d852-481d-bfd1-677c20f97427",
+         "readOnly":false,
+         "eventType":"AwsApiCall",
+         "managementEvent":true,
+         "recipientAccountId":"123456789033",
+         "eventCategory":"Management",
+         "sessionCredentialFromConsole":"true"         
 }
 ```
 
-
 ### Sample Query
 
-```sql title="Top 10 users"
-_sourceCategory=*cloudtrail* "\"eventsource\":\"sqs.amazonaws.com\""
-| json "eventSource" nodrop
-| json "userIdentity.type" as type nodrop
-| json "userIdentity.arn" as arn nodrop
-| json "userName" nodrop
-| json "eventName" nodrop
-| where eventSource="sqs.amazonaws.com"
-| parse field=arn ":assumed-role/*" as user
-| if (isEmpty(userName), user, userName) as user
-| count as eventCount by user
-| top 10 user by eventCount
+**Messages Received (Metric based)**:
+metric=NumberOfMessagesReceived Statistic=Sum account=* region=* namespace=* queuename=* | sum by account, region, namespace, queuename
+
+```sql title="Top 10 users (CloudTrail Log based)"
```

Comment:
Here 'Top 10 users (CloudTrail Log based):' should be present. I don't see it in the md file.